prow/monitoring/prow_prometheus.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prow
+  namespace: prow-monitoring
+spec:
+  replicas: 2
+  storage:
+    volumeClaimTemplate:
+      metadata:
+        name: prometheus
+      spec:
+        accessModes:
+          - "ReadWriteOnce"
+        storageClassName: "standard"
+        resources:
+          requests:
+            storage: 100Gi
+  retention: "30d"
+  serviceAccountName: prometheus-prow
+  alerting:
+    alertmanagers:
+      - namespace: prow-monitoring
+        name: alertmanager
+        port: web
+  enableAdminAPI: false
+  ruleSelector:
+    matchLabels:
+      role: alert-rules
+      prometheus: prow
+  serviceMonitorSelector:
+    matchExpressions:
+      - key: prow-app
+        operator: Exists
+  version: v2.7.1
+  baseImage: openshift/prometheus
```

Comment:
This needs to be a fully qualified pull spec

Reply:
`docker.io/prom/prometheus`

Reply:
Done